### ISSUE TITLE
WET4 - Same files and folders in multiple places

### ISSUE THREAD

**Issue by annpeiying** [wet-boew/wet-boew#7968]
Hi,

The latest version WET 4 seems to have same files and folders in multiple places :

themes-dist-4.0.24-theme-gcwu-fegc.zip\themes-dist-4.0.24-theme-gcwu-fegc\js\jquery\2.1.4
themes-dist-4.0.24-theme-gcwu-fegc.zip\themes-dist-4.0.24-theme-gcwu-fegc\js\jquery\1.11.1

themes-dist-4.0.24-theme-gcwu-fegc.zip\themes-dist-4.0.24-theme-gcwu-fegc\wet-boew\js\jquery\2.1.4
themes-dist-4.0.24-theme-gcwu-fegc.zip\themes-dist-4.0.24-theme-gcwu-fegc\wet-boew\js\jquery\1.11.1\

Would you please let me know why these files need to be in more than one places?

Thank you
-Anne

**Comment by RobJohnston** [comment]
If you view the source of [a sample page](http://wet-boew.github.io/themes-dist/theme-gcwu-fegc/content-en.html), you can see that jQuery 1.11.1 is loaded if the browser is an IE version before 9 and 2.1.4 otherwise.

The duplication of both is, I believe, due to historical reasons... backward compatibility for sites that don't have a wet-boew folder.

**Comment by nschonni** [comment]
It looks like the push to the CDN target is copying too much

**Comment by nschonni** [comment]
@LaurentGoderre I went back a few releases to 4.12 (randomly) and it doesn't have this issue, so it is a more recent change to the build setup

**Comment by LaurentGoderre** [comment]
@nschonni She isnt referencing the cdn but the dist though. The cdn was introduced in 4.0.16

**Comment by annpeiying** [comment]
Yes, I used the "dist" not from the CDN because there was a need for using the internal theme (i.e JS and CSS) .
